tc/core/polyhedral/cuda/memory_promotion_heuristic.cc

```diff
@@ -526,6 +526,21 @@ void promoteToSharedGreedy(
     scop.insertSyncsAroundCopies(bandNode);
   }
 }
+
+// Returns a union of all mapping filters to "MappingType" in "scop".
```

Comment:
Nit: I think we either use imperative ("Return") or an explicit subject ("This function returns")

The union is unambiguously defined, so it should be "the union".

It doesn't seems like you are actually taking any union, though...

Reply:
```
(tc_build) zinenko@citrouille:~/TensorComprehensions$ git grep "Returns " | wc -l
25
(tc_build) zinenko@citrouille:~/TensorComprehensions$ git grep "Return " | wc -l                                          
43
```
With such balance, I don't this there is even an implicit convention.